pkg/cert/truststore.go

```diff
@@ -201,6 +203,10 @@ func (m fileTrustStore) Verify(cert *x509.Certificate, moment time.Time) error {
 	return err
 }
 
+func (m fileTrustStore) VerifiedChain(cert *x509.Certificate, moment time.Time) ([][]*x509.Certificate, error) {
```

Comment:
I think you will need to add the intermediate certs to the VerifyOptions as well.

Reply:
With the new structure there are no intermediates. There's a Vendor CA (root) and the signing cert.